pkgs/development/tools/makerpm/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, zlib, libarchive, openssl }: 
+
+stdenv.mkDerivation { 
+  name = "makerpm-1.0";
+  version = "1.0";
+
+  builder = ./builder.sh;
```

Comment:
I don't really see a reason to use a builder here, instead of [overriding the installPhase](http://nixos.org/nixpkgs/manual/#sec-using-stdenv).